client/trino-cli/src/main/java/io/trino/cli/Trino.java

```diff
@@ -53,7 +53,10 @@ public static CommandLine createCommandLine(Object command)
                 .registerConverter(HostAndPort.class, HostAndPort::fromString)
                 .registerConverter(Duration.class, Duration::valueOf);
 
-        getConfigFile().ifPresent(file -> commandLine.setDefaultValueProvider(new CommandLine.PropertiesDefaultProvider(file)));
+        getConfigFile().ifPresent(file -> {
```

Comment:
Why would we warn here. If we treat it as a end-user feature we should not discourage user from using it.